sky/tools/create_macos_framework.py

```diff
@@ -220,7 +220,7 @@ def zip_framework(dst, args):
 def zip_xcframework_archive(dst):
   filepath_with_entitlements = ''
   filepath_without_entitlements = (
-      'FlutterMacOS.xcframework/macos-arm64_x84_64/'
+      'FlutterMacOS.xcframework/macos-arm64_x86_64/FlutterMacOS.framework/Versions/A/FlutterMacOS'
```

Comment:
Still need to remove the `/FlutterMacOS.framework/Versions/A/FlutterMacOS`, that's continued on the next line (line 224)

```suggestion
      'FlutterMacOS.xcframework/macos-arm64_x86_64/'
```

Reply:
Done. Ah so there is only 1 binary in the xcframework zip file?

Reply:
I think you have to keep it on 2 lines to pass the formatting linter

Reply:
> Done. Ah so there is only 1 binary in the xcframework zip file?

Yes, there's only `FlutterMacOS.xcframework/macos-arm64_x86_64/FlutterMacOS.framework/Versions/A/FlutterMacOS`

Reply:
I think it needs to looks like this or linter may get mad about the line being too long

```python
filepath_without_entitlements = (
      'FlutterMacOS.xcframework/macos-arm64_x86_64/'
      'FlutterMacOS.framework/Versions/A/FlutterMacOS'
  )
```

Reply:
Ah I see what you are saying now, done